src/main/resources/org/jenkinsci/plugins/configfiles/ConfigFilesManagement/selectprovider.properties

```diff
@@ -18,4 +18,4 @@
 
 selectprovider=Select the file type you want to create
 filetype=Type
-
+configIdSuffixDescription=Config file ID is prefixed by the provider name (e.g. "org.jenkinsci.plugins.configfiles.maven.MavenSettingsConfig") and suffixed by the given "ID suffix" or by a random number if no "ID Suffix" is specified

```

Comment:
BTW this still seems crazy. Is there no way for the whole ID to be user-specified? Why does the provider name need to be there?